homeassistant/components/logbook/__init__.py

```diff
@@ -300,27 +300,7 @@ def humanify(hass, events, entity_attr_cache, context_lookup):
                 start_stop_events[event.time_fired_minute] = 2
 
         # Yield entries
-        external_events = hass.data.get(DOMAIN, {})
         for event in events_batch:
-            if event.event_type in external_events:
```

Comment:
No change here, just a re-order since this is a tight loop and `EVENT_STATE_CHANGED` almost always comes first